index.html

```diff
@@ -0,0 +1,64 @@
+<html>
+    <head>
+        <title>Apple</title>
+        <link rel="stylesheet" href="./main.css" type="text/css">
+    </head>
+    <body>
+        <main class="container">
+            <section class="m-img">
+                    <nav>
+                    <ul>
+                        <li id="apple-logo"></li>
+                        <li id="store"></li>
+                        <li id="mac"></li>
+                        <li id="ipod"></li>
+                        <li id="iphone"></li>
+                        <li id="ipad"></li>
+                        <li id="itunes"></li>
+                        <li id="support"></li>
+                        <li id="search"><input type="text"></li>
+                    </ul>        
+                </nav>
+                <header>
+                    <div class="heading"></div>
+                    <div class="sub-heading"></div>
```

Comment:
Nice job loading the text images using the div tag. But all of these 3 texts must be a link. Put each div inside an `a` tag. Three `a` and three `div`.